.github/workflows/ci-cabal.yaml

```diff
@@ -186,7 +186,7 @@ jobs:
       id: comment-body
       run: |
         # Drop first 5 header lines and demote headlines one level
-        body="$(cat artifact/transaction-cost.md | sed '1,5d;s/^#/##/')"
+        body="$(cat <(cat artifact/transaction-cost.md | sed '1,5d;s/^#/##/') <(cat artifact/end-to-end-benchmarks.md | sed '1,5d;s/^#/##/') | grep -v '^:::')"
```

Comment:
I would prefer having two separate comments on the PR. One for the on-chain metrics, one for the off-chain metrics?